Cargo.toml

```diff
@@ -30,17 +30,18 @@ required-features = ["tfidf", "textrank"]
 regex = "1.0"
 lazy_static = "1.0"
 phf = "0.11"
-hashbrown = { version = "0.12", default-features = false, features = ["inline-more"] }
 cedarwood = "0.4"
 ordered-float = { version = "3.0", optional = true }
+once_cell = "1"
```

Comment:
There is no point in use both `lazy_static` and `once_cell`, please also replace `lazy_static` with `once_cell`